go.mod

```diff
@@ -71,6 +71,7 @@ require (
 	google.golang.org/protobuf v1.31.0
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/yaml.v3 v3.0.1
+	k8s.io/kubernetes v1.13.0
```

Comment:
Afaik k8s deps were removed intentionally, this PR is adding it back. @Luap99 @umohnani8 PTAL

Reply:
It's only adding k8s.io/kubernetes/third_party/forked/golang/expansion as dependency which I think is tolerable.

Reply:
The K8s deps were only dropped because they added ridiculous amount of bloat that was never used in podman, i.e. the full RPC code. The code imported into podman (pkg/k8s.io) is heavily modified, mainly dropped all the generated protobuf code that was responsible for most of the binary size.

If this is only using a single package without big dependencies it is totally fine to use that. And I much rather prefer that in order to get potential updates.